+ java config for service layer

aaabramov opened the pull request:
- \- xml config for service layer

skyfenko commented on web/src/main/resources/spring/web-common-ctx.xml:
```diff
@@ -56,5 +56,9 @@
 
     <context:component-scan
-            base-package="edu.softserve.zoo.controller.rest, edu.softserve.zoo.converter, edu.softserve.zoo.advice"/>
+            base-package="edu.softserve.zoo.controller.rest,
```
assuming that is temporary and you wouldn't use such an approach to pull a module lying below

skyfenko commented on service/src/main/java/edu/softserve/zoo/service/config/ServiceTestConfig.java:
```diff
@@ -0,0 +1,16 @@
+ * @author Andrii Abramov on 6/13/16.
+ */
+@Configuration
+@Profile("test")
```
using profile on class basis is too poor approach to provide flexibility. use profile on beans instead

aaabramov committed:
+ normalized configurations
+ profiles constants
+ @Profile on @Bean's level.
+ common Service Config
+ common persistence config
+ moved tasks specs package
